portfolio/src/main/webapp/script.js

```diff
@@ -55,19 +56,31 @@ function createListElement(text) {
   return liElement;
 }
 
+/**
+ * Loads entities from the datastore and adds them to the DOM.
+ */
+function loadCommentsFromDatastore() {
+  let value = document.getElementById('num-comments').value;  
+  fetch(`/list-comments?num-comments=${value}`).then(response => response.json()).then((comments) => {
+    console.log(comments)
+    const commentElement = document.getElementById('comments-container');
+    commentElement.innerHTML = '';
+    comments.forEach((indComment) => {
+      commentElement.appendChild(createListElement(indComment.name + " says: " + indComment.content));
+    })
+  });
+}
+
 /**
  * Fetches comments from the servers and adds them to the DOM.
  */
 function getComments() {
```

Comment:
Is this function still used somewhere? If not, we can remove it.

Reply:
This function is no longer used. It was used before the datastore was implemented. I will remove it.